app/src/main/java/com/osfans/trime/Composition.java

```diff
@@ -228,10 +228,12 @@ public void reset(Context context) {
     setMinHeight(config.getPixel("layout/min_height"));
     setMaxWidth(config.getPixel("layout/max_width"));
     setMaxHeight(config.getPixel("layout/max_height"));
-    int margin_x, margin_y;
+    int margin_x, margin_y, margin_bottom;
     margin_x = config.getPixel("layout/margin_x");
     margin_y = config.getPixel("layout/margin_y");
-    setPadding(margin_x, margin_y, margin_x, margin_y);
+    margin_bottom = config.getPixel("layout/margin_bottom",margin_y);
+    System.out.println("Composition setPadding=" + margin_x+", "+margin_y+", "+margin_bottom);
```

Comment:
Use android.util.Log for logging?

Reply:
for logging, not necessary. I have remove these system.out.print() just now